reference/docs-conceptual/learn/shell/running-commands.md

```diff
@@ -0,0 +1,163 @@
+`$?` is set to `$true`.
+
+However, this changed in PowerShell 7.2. Error records redirected from native commands, like when
+using redirection operators (`2>&1`), are not written to PowerShell's `$Error` variable and the
+preference variable `$ErrorActionPreference` does not affect the redirected output.
+
+Many native commands write to `stderr` as an alternative stream for additional information. This
+behavior can cause confusion in PowerShell when looking through errors and the additional output
+information can be lost if `$ErrorActionPreference` is set to a state that mutes the output.
+
+PowerShell 7.3 added a new experimental feature `PSNativeCommandErrorActionPreference` that allows
+you to control whether output to `stderr` is treated as an error. For more information, see
+[PSNativeCommandErrorActionPreference][6].
+
+## Running PowerShell commands
+
+As previous noted, PowerShell commands are are known as cmdlets. Cmdlets are collected into
+PowerShell modules that can be loaded on demand. Cmdlets can be written in any compiled .NET
+language or using the PowerShell scripting language itself.
+
+### PowerShell commands that run other commands
+
+The PowerShell **call operator** (`&`) lets you run commands that are stored in variables and
+represented by strings or script blocks. You can use this to run any native command or PowerShell
+command. This is useful in a script when you need to dynamically construct the command-line
+parameters for a native command. For more information, see the [call operator][7].
+
+The `Start-Process` cmdlet can be used to run native commands, but should only be used when you need
+to control how the command is executed. The cmdlet has parameters to support the following
+scenarios:
+
+- Run a command using different credentials
+- Hide the console window created by the new process
+- Redirect **stdin**, **stdout**, and **stderr** streams
+- Use a different working directory for the command
+
+The following example runs the native command `sort.exe` with redirected input and output streams.
+
+```powershell
+$processOptions = @{
+    FilePath = "sort.exe"
+    RedirectStandardInput = "TestSort.txt"
+    RedirectStandardOutput = "Sorted.txt"
+    RedirectStandardError = "SortError.txt"
+    UseNewEnvironment = $true
+}
+Start-Process @processOptions
+```
+
+For more information, see [Start-Process][8].
+
+On Windows, the `Invoke-Item` command performs the default action for the specified item. For
+example, it runs an executable file or opens a document file using the application associated with
+the document file type. The default action depends on the type of item and is resolved by the
+PowerShell provider that provides access to the item.
+
+The following example opens the PowerShell source code respoitory in your default web browser.
+
+```powershell
+Invoke-Item https://github.com/PowerShell/PowerShell
```

Comment:
Nit: No parameter here. Is that to simplify things for the reader?

Reply:
Yes. The parameter would be `-Path`. That could be confusing in this context since it is not a file path.